pages/windows/assoc.md

```diff
@@ -0,0 +1,19 @@
+# assoc
+
+> Displays or modifies file name extension associations. If used without parameters, assoc displays a list of all the current file name extension associations.
```

Comment:
If use it without parameters, assoc displays a list of all extension associations.